PROTO_tests/tests/operator_test.py

```diff
@@ -0,0 +1,774 @@
+                        f"Cannot detect return dtype of ak result: {akres} (np result: {npres})"
+                    )
+                    results["both_implement"].append((expression, str(akres), False, True, False))
+                    continue
+
+                if akrestype != npres.dtype:
+                    restypes = f"{npres.dtype}(np) vs. {akrestype}(ak)"
+                    results["both_implement"].append((expression, restypes, False, True, False))
+                    continue
+                try:
+                    akasnp = akres.to_ndarray()
+                except Exception:
+                    warnings.warn(f"Could not convert to ndarray: {akres}")
+                    results["both_implement"].append((expression, str(akres), True, False, False))
+                    continue
+                if not np.allclose(akasnp, npres, equal_nan=True):
+                    res = f"np: {npres}\nak: {akasnp}"
+                    results["both_implement"].append((expression, res, False, False, True))
+                    continue
+                # Finally, both numpy and arkouda agree on result
+                results["both_implement"].append((expression, "", False, False, False))
+
+        print(f'# ops not implemented by numpy or arkouda: {len(results["neither_implement"])}')
+        if verbose:
+            for expression, err in results["neither_implement"]:
+                print(expression)
+        print(f'# ops implemented by numpy but not arkouda: {len(results["numpy_minus_arkouda"])}')
+        if verbose:
+            for expression, err, flag in results["numpy_minus_arkouda"]:
+                print(expression)
+        print(f'# ops implemented by arkouda but not numpy: {len(results["arkouda_minus_numpy"])}')
+        if verbose:
+            for expression, res, flag in results["arkouda_minus_numpy"]:
+                print(expression, " -> ", res)
+        nboth = len(results["both_implement"])
+        print(f"# ops implemented by both: {nboth}")
+        matches = 0
+        execerrors = []
+        dtypeerrors = []
+        valueerrors = []
+        for expression, res, ex, dt, val in results["both_implement"]:
+            matches += not any((ex, dt, val))
+            if ex:
+                execerrors.append((expression, res))
+            if dt:
+                dtypeerrors.append((expression, res))
+            if val:
+                valueerrors.append((expression, res))
+        print(f"  Matching results:         {matches} / {nboth}")
+        print(f"  Arkouda execution errors: {len(execerrors)} / {nboth}")
+        if verbose:
+            print("\n".join(map(": ".join, execerrors)))
+        print(f"  Dtype mismatches:         {len(dtypeerrors)} / {nboth}")
+        if verbose:
+            print("\n".join(map(": ".join, dtypeerrors)))
+        print(f"  Value mismatches:         {len(valueerrors)} / {nboth}")
+        if verbose:
+            print("\n".join(map(": ".join, valueerrors)))
```

Comment:
Again, we could use a single `if verbose` block here.

Reply:
I believe this the same deal, we only wanna print the mismatches with the appropriate header

Reply:
Same response as above.